src/azure-cli/azure/cli/command_modules/deploymentmanager/custom.py

```diff
@@ -73,6 +77,15 @@ def cli_artifact_source_update(
     return instance
 
 
+def cli_artifact_sources_list(
```

Comment:
When you specify client_factory in https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/commands.py#L24, you can call the operations directly. You don't need to redefine custom function any more.